app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FeedbacksController < ApplicationController
+  before_action :set_feedback, only: %i[show edit update]
+
+  def index; end
+
+  def show
+    authorize @feedback
+  end
+
+  def new; end
+
+  def edit
+    authorize @feedback
+  end
+
+  def update
+    authorize @feedback
+
+    if @feedback.update_with_json_answers(params)
```

Comment:
this should pass in the status we want to set (or it should just set the status to completed automatically).

Reply:
hmm, i set the status inside the if block to save if the status is updated. This is in line 20